Dockerfile

```diff
@@ -6,6 +6,9 @@ ARG VPNKIT_VERSION=0.5.0
 ARG PASST_VERSION=2025_02_17.a1e48a0
 ARG DOCKER_VERSION=28.0.1
 ARG DOCKER_CHANNEL=stable
+# also tested with aarch64
+ARG ARCH=x86_64
```

Comment:
This should use `TARGETARCH` to support `docker build --platform`

https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/

Reply:
Or maybe just checking `$(uname -m)` may suffice, then no need to use `ARG`